integration/util/scenario/token.js

```diff
@@ -205,6 +206,7 @@ function tokenInfoMap(ctx) {
 
   return {
     zrx: {
+      chain: 'matic',
```

Comment:
We do use `zrx` in tests, so we probably should make a new token / set of tokens for matic.

Reply:
found a problem with `setSupplyCap` when I addressed this so, yay :)